core/src/hir/attrs.rs

```diff
@@ -81,6 +81,9 @@ pub struct DemoInfo {
     /// `#[diplomat::demo(external)]` represents an item that we will not evaluate, and should be passed to the rendering engine to provide.
     pub external: bool,
 
+    /// `#[diplomat::demo(custom_func = "/file/name/here.mjs")]` represents a user defined function that they define themselves in JS.
```

Comment:
could you document more on what this does? I'm not really understanding the effect of setting this right now